### PR TITLE
network matching

### PR DESCRIPTION
**Pull request by miajeffris**
Uploaded filtered GRNdb datasets (all interactions + high confidence interactions only), uploaded zip file for RegNetwork data, wrote graph functions for GRNdb datasets, deleted an erroneous duplicate set_nodes() function in graph.py.

### CODE REVIEW

File: de/grndb_network.py [hunk 1]
```diff
@@ -0,0 +1,118 @@
+"""Contains functions for creating w matrix and directed graph for GRNdb dataset."""
+from os.path import join, dirname
+import networkx as nx
+import pandas as pd
+import numpy as np
+import matplotlib.patches as mpatches
+import matplotlib.lines as mlines
```
Comment (Farnazmdi) [inline de/grndb_network.py:7]: Have you used `mlines` anywhere? if not, please remove this line.

Comment (miajeffris) [reply de/grndb_network.py:7]: You're right, I removed it!

**Comment by Farnazmdi** [comment]
Great! Could you post a screenshot of the figure3 to see what it looks like now?

**Comment by Farnazmdi** [comment]
Hmm.. I thought nodes with higher connectivity will be larger based on the PageRank. Why is, for example, GATA4, larger than KDM2A?

**Comment by miajeffris** [comment]
Yeah I was wondering that too. I did some research about the pagerank algorithm and I'm wondering if it just has to do with the "quality" of the connections. The only thing I changed about the `pagerank()` function was not dividing by zero, like we talked about. So nodes that don't point to anything might have pagerank values that are a little unexpected, though I'm not totally sure how this would change them drastically. Basically, where some columns have normalized values adding up to 1, others have just zeroes before going through the next steps in `pagerank()`.

**Comment by Farnazmdi** [comment]
@aarmey Could you please review/merge this branch? The `pagerank` function is not updated in this, but we can update that after we merge Mia's new branch.

**Review by aarmey — APPROVED** [review]

**Pull request merged**